symplyphysics/laws/thermodynamics/work_is_integral_of_pressure_over_volume.py

```diff
@@ -1,36 +1,83 @@
+"""
+Work is integral of pressure over volume
+========================================
+
+The pressure-volume work is the work done by a thermodynamic system when the volume of the
+system changes.
+
+**Conditions:**
+
+#. The process is reversible or quasi-static.
+#. The system is closed.
+"""
+
 from sympy import Eq, Integral, Point2D, solve, symbols, Function as SymFunction
 from symplyphysics import (
     units,
     Quantity,
     Symbol,
     Function,
-    print_expression,
     validate_input,
     validate_output,
 )
 from symplyphysics.core.expr_comparisons import expr_equals
 from symplyphysics.core.geometry.line import two_point_function
 from symplyphysics.laws.thermodynamics import infinitesimal_work_in_quasistatic_process as work_law
 
-# Description
-## The pressure-volume work occurs when the volume of a system changes.
-
-# Law: W = Integral(p(V), V)
-## W - work done by gas
-## p - gas pressure
-## V - gas volume
+work = Symbol("work", units.energy)
+"""
+Pressure-volume work done by the system.
 
-# Conditions
-## - The process is reversible or quasi-static.
-## - The system is closed.
+Symbol:
+    :code:`W`
+"""
 
-work = Symbol("work", units.energy)
 pressure = Function("pressure", units.pressure)
+"""
+Pressure inside the system as a function of volume.
+
+Symbol:
+    :code:`p(V)`
+"""
+
 volume = Symbol("volume", units.volume)
-volume_before = Symbol("volume_before", units.volume)
-volume_after = Symbol("volume_after", units.volume)
+"""
+Volume of the system.
+
+Symbol:
+    :code:`V`
+"""
 
-law = Eq(work, Integral(pressure(volume), (volume, volume_before, volume_after)))
+initial_volume = Symbol("initial_volume", units.volume)
+r"""
+Initial volume of the system.
+
+Symbol:
+    :code:`V_0`
+
+Latex:
+    :math:`V_0`
+"""
+
+final_volume = Symbol("final_volume", units.volume)
+r"""
+Final volume of the system.
+
+Symbol:
+    :code:`V_1`
+
+Latex:
+    :math:`V_1`
+"""
+
+law = Eq(work, Integral(pressure(volume), (volume, initial_volume, final_volume)))
+r"""
+:code:`W = Integral(p(V), (V, V_1, V_0))`
```

Comment:
I think initial integration limit should come first: `Integral(p(V), (V, V_0, V_1))`